src/main/java/de/hpi/bpt/chimera/execution/data/DataManager.java

```diff
@@ -559,6 +511,29 @@ public DataObject getDataObjectById(String dataObjectId) {
 				.filter(d -> !d.isLocked())
 				.collect(Collectors.toSet());
 	}
+
+	/**
+	 * Get all Dataobjects of Dataclasses that are in the post condition and that are in a
```

Comment:
that are in the post condition and that are in a -> that are in the post condition and in a ...
Bin mir natürlich nicht sicher hört sich aber besser für mich besser an